imap_processing/codice/codice_l1a.py

```diff
@@ -309,49 +315,49 @@ def unpack_science_data(self, science_values: str) -> None:
         science_values : str
             A string of binary data representing the science values of the data.
         """
-        self.compression_algorithm = constants.LO_COMPRESSION_ID_LOOKUP[self.view_id]
+        compression_algorithm = constants.LO_COMPRESSION_ID_LOOKUP[self.view_id]
```

Comment:
When I originally wrote this, I thought that I might be using the `compression_algorithm` in various places in the module, but turns out I only need it here, so no need to make it a class attribute.